packages/shared/src/components/filters/ContentTypesFilter.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from 'react';
+import { FilterSwitch } from './FilterSwitch';
+import { useAdvancedSettings } from '../../hooks/feed';
+import useFeedSettings from '../../hooks/useFeedSettings';
+
+export function ContentTypesFilter(): ReactElement {
+  const { advancedSettings, isLoading } = useFeedSettings();
+  const { selectedSettings, onToggleSettings } = useAdvancedSettings();
+  const videos = advancedSettings.find(({ title }) => title === 'Videos');
```

Comment:
Should we not match them on groups perhaps?
Just wondering what happens now if we introduce `Collections` for instance as we filterable type?

Reply:
Guess for now we can leave it, if it's out of scope that's fine.
Something to note for a future change.

Just wondering what we need the group for now then?

Reply:
Yeah, maybe I need to add a comment on the implementation.

In order for the group to display, we should add the row in our DB.